src/bindings/python/src/openvino/offline_transformations/__init__.py

```diff
@@ -3,20 +3,108 @@
 # SPDX-License-Identifier: Apache-2.0
 
 # flake8: noqa
+# type: ignore
 
-from openvino.utils import add_openvino_libs_to_path
+import warnings
+
+warnings.warn(message="The module is private and following namespace "
+                      "`offline_transformations` will be removed in "
+                      "the future, use `openvino.runtime.passes` instead!",
+              category=FutureWarning)
+
+from openvino.utils import add_openvino_libs_to_path, deprecated
 add_openvino_libs_to_path()
 
 from openvino.pyopenvino import get_version
+from openvino.pyopenvino import serialize as _base_serialize
+import openvino.pyopenvino._offline_transformations as _base
+
 __version__ = get_version()
 
-from openvino.pyopenvino.offline_transformations import apply_moc_transformations
-from openvino.pyopenvino.offline_transformations import apply_moc_legacy_transformations
-from openvino.pyopenvino.offline_transformations import apply_pot_transformations
-from openvino.pyopenvino.offline_transformations import apply_low_latency_transformation
-from openvino.pyopenvino.offline_transformations import apply_pruning_transformation
-from openvino.pyopenvino.offline_transformations import generate_mapping_file
-from openvino.pyopenvino.offline_transformations import apply_make_stateful_transformation
-from openvino.pyopenvino.offline_transformations import compress_model_transformation
-from openvino.pyopenvino.offline_transformations import compress_quantize_weights_transformation
-from openvino.pyopenvino.offline_transformations import convert_sequence_to_tensor_iterator_transformation
+
+@deprecated(version="2023.1",
+            message="The module is private and following namespace "
+                    "`offline_transformations` will be removed in "
+                    "the future, use `openvino.runtime.passes` instead!")
+def serialize(model, xml_path, bin_path, version):
+    _base_serialize(model, xml_path, bin_path, version)
+
+
+@deprecated(version="2023.1",
+            message="The module is private and following namespace "
+                    "`offline_transformations` will be removed in "
+                    "the future, use `openvino.runtime.passes` instead!")
```

Comment:
Does user know of a set of transformations from `offline_transformations` to be registered in order to repeat `apply_moc_transformations`? The same question is for other `offline_transformation` routines.

Reply:
Message has been changed.